src/Stryker.Core/Stryker.Core.UnitTest/StrykerRunResultTests.cs

```diff
@@ -0,0 +1,43 @@
+﻿using Shouldly;
+using Stryker.Core.Options;
+using Xunit;
+
+namespace Stryker.Core.UnitTest
+{
+    public class StrykerRunResultTests
+    {
+        [Theory]
+        [InlineData(1, 80)]
+        [InlineData(0.5, 50)]
+        [InlineData(0.1, 0)]
+        public void ScoreIsLowerThanThresholdBreak_ShouldReturnFalseWhen(double mutationScore, int thresholdBreak)
+        {
+            // Arrange
+            var options = new StrykerOptions(thresholdHigh: 100, thresholdLow: 100, thresholdBreak: thresholdBreak);
+            var runResult = new StrykerRunResult(options, mutationScore);
+
+            // Act
+            var scoreIsLowerThanThresholdBreak = runResult.ScoreIsLowerThanThresholdBreak();
+
+            // Assert
+            scoreIsLowerThanThresholdBreak.ShouldBeFalse("because the mutation score is higher than the threshold break");
```

Comment:
```suggestion
            scoreIsLowerThanThresholdBreak.ShouldBeFalse("because the mutation score is higher than or equal to the threshold break");
```